lib/ansible/modules/debconf.py

```diff
@@ -217,15 +219,19 @@ def main():
             if vtype == 'boolean':
                 value = to_text(value).lower()
                 existing = to_text(prev[question]).lower()
-
-            if vtype == 'password':
+            elif vtype == 'password':
                 existing = get_password_value(module, pkg, question, vtype)
+            elif vtype == 'multiselect' and isinstance(value, list):
+                value = sorted(value)
+                existing = sorted([i.strip() for i in existing.split(",")])
 
             if value != existing:
                 changed = True
 
     if changed:
         if not module.check_mode:
+            if vtype == 'multiselect' and isinstance(value, list):
+                value = ", ".join(value)
```

Comment:
Since we cannot use the argument parser to validate the elements, we should wrap this in a `try/except` to avoid a traceback if someone passed something like an `int`.